src/main.ts

```diff
@@ -51,403 +34,82 @@ if (_PACKAGING !== "0") {
 // Logger
 const debug = debug_("readium-desktop:main");
 
-// Parse command line
-const processArgs = yargs.argv;
-
+// Global
```

Comment:
In this PR, `main.ts` gets divided into several constituent parts, which I think is a good idea (makes the code clearer to read and maintain). 

However, we now have to be **very** careful before merging, because there have been changes to `main.ts` pushed to `develop` since this PR was started:
https://github.com/readium/readium-desktop/commits/develop/src/main.ts

For example, the `console` disabling:
https://github.com/readium/readium-desktop/commit/c2296e761b40da0b779bb458b033a3df68144e14#diff-8cfead41d88ad47d44509a8ab0a109ad
...should show-up "automatically" just above this PR comment (when looking at the code diff), once `develop` is merged back into this PR's branch.

BUT, potential changes made to other parts of `main.ts` that are now separated into `createWindow.ts` or `init.ts` will not automatically be "ported" there. For example, the `i18next` locale sync is now in `init.ts`, but it is behind the tip of the `develop` branch:
https://github.com/readium/readium-desktop/commit/ffe98cb490bbfe89ff0200951dd07112deaa93fc#diff-8cfead41d88ad47d44509a8ab0a109ad
(search for the string `const lang = Object.keys(AvailableLanguages).find((l) => l === loc) || "en";` in this PR code diff)

Having analysed the Git commit history of `main.ts`, I think there are no other instances of breaking changes. @panaC what do you think?